config/build/README.md

```diff
@@ -0,0 +1,3 @@
+# persus-build
+
+TODO(FEI-3759): setup webpack to build perseus
```

Comment:
I noticed this PR uses Rollup. Is this comment still accurate?

Reply:
🤦‍♂️ I'll update this.